ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -474,13 +474,13 @@ Rectangle {
                             }
                         }
                     }
-
                     insertInTextInput(d.copyTextStart, d.copiedTextFormatted)
                 } else {
                     d.copiedTextPlain = ""
                     d.copiedTextFormatted = ""
                     d.copiedMentionsPos = []
-                    messageInputField.insert(d.copyTextStart, "<div style='white-space: pre-wrap'>" + Utils.escapeHtml(QClipboardProxy.text) + "</div>") // preserve formatting
+                    messageInputField.insert(d.copyTextStart, (StatusQUtils.Emoji.hasEmoji(QClipboardProxy.html) ? //Utils.escapeHtml(
```

Comment:
what's the commented out chunk? also, this will likely destroy pasting code snippets :/